docker-compose.override.yml

```diff
@@ -7,7 +9,7 @@ services:
       - "bats-tests:host-gateway"
   otel-agent:
     ports:
-      - "4318:4318" # http receiver
+      - "4318:4318" #! http receiver
```

Comment:
what does the `!` do?

Reply:
This is for `ytt` to interpret it as a comment. Unfortunately `ytt` does not accept legal yaml comments starting with `#` only.